src/api/storage-pools.tsx

```diff
@@ -71,6 +107,23 @@ export const updateStoragePool = (pool: LxdStoragePool, project: string) => {
   });
 };
 
+export const updateClusteredPool = (
+  pool: Partial<LxdStoragePool>,
+  project: string,
+  clusterMembers: LxdClusterMember[],
+): Promise<void> => {
```

Comment:
Is there a particular reason why the return type is specified here but not for the `createClusteredPool` above?

Reply:
A leftover from development